app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -27,16 +28,19 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
     val message = SingleLiveEvent<String>()
     val event = MutableLiveData<Event>()
     var attendee = MutableLiveData<User>()
-    val tickets = MutableLiveData<List<Ticket>>()
     var paymentSelectorVisibility = MutableLiveData<Boolean>()
     var totalAmount = MutableLiveData<Float>()
     var totalQty = MutableLiveData<Int>()
     val qtyList = MutableLiveData<ArrayList<Int>>()
     val month = ArrayList<String>()
     val year = ArrayList<String>()
+    var country: String? = null
+    var paymentOption: String? = null
     val cardType = ArrayList<String>()
     var orderIdentifier: String? = null
     var paymentCompleted = MutableLiveData<Boolean>()
+    var attendees = MutableLiveData<MutableList<Attendee>>()
+    var tickets = MutableLiveData<MutableList<Ticket>>()
```

Comment:
Where are we using these in fragment and if we're not, then why are these MutableLiveData?